tests/dragonfly/snapshot_test.py

```diff
@@ -124,6 +126,32 @@ async def test_dbfilenames(
             assert await StaticSeeder.capture(client) == start_capture
 
 
+@pytest.mark.asyncio
+@dfly_args({**BASIC_ARGS, "proactor_threads": 4, "dbfilename": "test-redis-load-rdb"})
+async def test_redis_load_snapshot(
+    async_client: aioredis.Redis, df_server, redis_local_server: RedisServer, tmp_dir: Path
+):
+    """
+    Test redis server loading dragonfly snapshot rdb format
+    """
+    await StaticSeeder(
+        **LIGHTWEIGHT_SEEDER_ARGS, types=["STRING", "LIST", "SET", "HASH", "ZSET"]
+    ).run(async_client)
+
+    await async_client.execute_command("SAVE", "rdb")
+    dbsize = await async_client.dbsize()
+
+    await async_client.connection_pool.disconnect()
+    df_server.stop()
+
+    redis_local_server.start(dir=tmp_dir, dbfilename="test-redis-load-rdb.rdb")
+    await asyncio.sleep(1)
+    c_master = aioredis.Redis(port=redis_local_server.port)
+    await c_master.ping()
+
+    assert await c_master.dbsize() == dbsize
```

Comment:
First redis compatibility test 👍🏻 

If more checks are needed, you can re-define `dragonfly.ihash` in redis. It's bit of a hassle, but doable 🙂 

```lua
local dragonfly = {}
function draognfly.ihash(hash, sort, ...)
  local res = redis.pcall(arg)
  -- if sort, sort res
  -- use any hash function to res... lua really doesn't have one :(
end
```

Reply:
I wanted to use the seeder capture but got error
redis.exceptions.ResponseError: Error compiling script (new function): user_script:1: unexpected symbol near '#'

I want the fix to go to 1.19, so I will merge this now and can investigate more if we can do a simple change to make this work

Reply:
@adiholden  you got an error with Redis, right?

Reply:
do you mean the script error? yes its in redis